bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -228,7 +228,17 @@ <h2 class="primary go">{{ _('Teach the Web') }}</h2>
           </a>
         </li>
       {% endif %}
-      {% if l10n_has_tag('fx10_independent') %}
+      {% if l10n_has_tag('firefox-private-browsing') %}
+        <li id="promo-10" class="item promo-large-portrait private-browsing" tabindex="0">
+          <h2 class="primary go">{{ _('Introducing the new Private Browsing') }}</h2>
+          <a class="panel-link" href="{{ url('firefox.developer') }}" data-promotion-id="promo-10" data-promotion-name="Introducing the new Private Browsing" data-promotion-class="promo-large-portrait" data-promotion-type="tile">
+            <div class="secondary">
+              <h3>{{ _('More protection.') }}<br>{{ _('The most privacy.') }}</h3>
+              <p class="more">{{ _('Try private browsing') }}</p>
```

Comment:
This should be `Try Private Browsing`, both based on copy deck and the landing page.